### DISCUSSION TITLE
MAJ automatique du taux versement transport

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2621] [Idées]
Faire un script de MAJ automatique du versement transport :

* Télécharger le fichier https://fichierdirect.declaration.urssaf.fr/static/tauxTransport.20191010.csv
* Créer une Action Github avec un cron (Hebdomadaire ? Autre périodicité ?) https://help.github.com/en/articles/workflow-syntax-for-github-actions#onschedule
* Si le fichier a été modifié, créer une PR automatiquement https://github.com/marketplace/actions/create-pull-request

**Comment by laem** [comment]
J'ai mis à jour le README du dépôt betagouv/taux-versement-transport pour clarifier le processus de mise à jour
